File: 1-two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        Map<Integer,Integer>mp=new HashMap<>();
        int n=nums.length;
        for(int i=0;i<n;i++){
            mp.put(nums[i],i);
        }
        for(int i=0;i<n;i++){
            int r=target-nums[i];
            if(mp.containsKey(r)&&mp.get(r)!=i)
            return new int[]{i,mp.get(r)};
        }
        return new int[]{};
    }
}

File: 1-two-sum/two-sum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        for(int j=0;j<nums.Length;j++){
            int t=target-nums[j];
            for(int i=j+1;i<nums.Length;i++){
                if(t==nums[i])
                return [i,j];
            }
        }
        return [0,0];
    }
}